### PR TITLE
Add test for different kafka version

### PR DESCRIPTION
**Pull request by bjoernhaeuser**
#### Pull Request (PR) description

Add a acceptance tests which should work with a different kafka version.

#### This Pull Request (PR) fixes the following issues

See comment in #257

### CODE REVIEW

**Comment by bjoernhaeuser** [comment]
@bastelfreak The test was already there, just updated it to a current kafka versiojn

File: manifests/params.pp [hunk 1]
```diff
@@ -20,8 +20,8 @@
   $config_dir     = '/opt/kafka/config'
   $bin_dir        = '/opt/kafka/bin'
   $log_dir        = '/var/log/kafka'
-  $mirror_url     = 'https://www.apache.org/dyn/closer.cgi?path='
-  $mirror_subpath = "/kafka/${version}"
+  $mirror_url     = 'https://www.apache.org/dyn/closer.lua?action=download&filename='
```
Comment (bastelfreak) [inline manifests/params.pp]: this looks like a bugfix that should be made in a separate pull request

Comment (bjoernhaeuser) [reply manifests/params.pp]: Yeah, well, in the end you are right, but without it does not work..

**Comment by bjoernhaeuser** [comment]
@bastelfreak Changed!
